frida_tools/lsd.py

```diff
@@ -19,9 +19,21 @@ def _start(self) -> None:
                 self._update_status(f"Failed to enumerate devices: {e}")
                 self._exit(1)
                 return
+            device_os = {}
+            device_name = {}
```

Comment:
Nit: These should be in the same order as they're displayed, so `device_name` followed by `device_os`.